File: Python/Array/Maximum Value of an Ordered Triplet I.py
class Solution:
    def maximumTripletValue(self, nums: List[int]) -> int:
        val = 0
        for i in range(len(nums)-2):
            for j in range(i+1,len(nums)-1):
                for k in range(j+1,len(nums)):
                    curr=(nums[i] - nums[j]) * nums[k]
                    if curr>val:
                        val=curr
        return val

File: Java/Array/Maximum Value of an Ordered Triplet I.java
class Solution {
    public long maximumTripletValue(int[] nums) {
        long val=0;
        for(int i=0;i<nums.length-2;i++){
            for(int j=i+1;j<nums.length-1;j++){
                for(int k=j+1;k<nums.length;k++){
                    long curr=(long)(nums[i] - nums[j]) * nums[k];
                    if(curr>val){
                        val=curr;
                    }
                }
            }
        }
        return val;
    }
}
